.github/workflows/node-ci.yml

```diff
@@ -74,9 +74,12 @@ jobs:
     strategy:
       matrix:
         os:
-          - [self-hosted, macOS, ARM64]
```

Comment:
This is macos-12 as well, and a lot faster - can it not run there?

Reply:
I can change it do that if it works. I was testing on my own fork so I don't think I have that there.

Reply:
Also, if this is compiled on arm, does this make it not work on other architectures? in the origional bring back node thread it was requested I use the older realease (when it was available) because otherwise it required libs you couldn't get on the older OS versions.

Reply:
I believe it's possibly true. The same could happen with these GitHub runners in the near future https://github.com/github/roadmap/issues/528

The thing is that M1's are much more economical to use for runners because they roughly have half the power draw and twice the compute, which is why they are a lot cheaper for us to self-host on aws as well: https://devdosvid.blog/2021/01/19/mac1.metal-and-mac2.metal-ec2-instances-user-experience/

I don't have a good enough overview of our aws budget yet to figure out if we can set up an x86 instance to make this build.